tap_helpscout/client.py

```diff
@@ -44,7 +47,14 @@ def __exit__(self, exception_type, exception_value, traceback):
                           max_tries=5,
                           factor=2)
     def get_access_token(self):
-        if self.__access_token is not None and self.__expires > datetime.utcnow():
+        # if tap is being executed in dev_mode then disable tap from creating new refresh and access tokens
+        if self.__dev_mode:
+            if self.__access_token:
+                return
+
+            raise Exception("Access token is missing, unable to authenticate in dev mode")
+
+        if self.__access_token and self.__expires > datetime.utcnow():
             return
```

Comment:
As per L#34, we are assuming access token is expired while running in non dev-mode then above condition looks redundant.